src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final int colorsCount = 3;
+
+    public String getRandomColor() {
+        return Color.values()[new Random().nextInt(colorsCount)].name();
```

Comment:
is it really neccesary to create new Random() each time when you call this method? think about extracting Random to class-level field and use it from this method